databases/week3/week3.sql

```diff
@@ -0,0 +1,166 @@
+CREATE DATABASE meal;
+use meal;
+
+CREATE TABLE meal (
+	id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    description TEXT (65535) NOT NULL,
+    location VARCHAR(255) NOT NULL,
+    `when` DATETIME NOT NULL,
+    max_reservation INT NOT NULL,
+    price DECIMAL (10, 2),
+    created_date DATE    
+);
+
+CREATE TABLE reservation (
+   id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+   number_of_guests INT NOT NULL,
+   meal_id INT NOT NULL,
+   created_date DATE,   
+   FOREIGN KEY (meal_id) references `meal`(id)
+);
+
+CREATE TABLE review (
+   id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+   title VARCHAR(255) NOT NULL,
+   description TEXT (65535) NOT NULL,
+   meal_id INT NOT NULL,
+   stars INT NOT NULL,
+   created_date DATE,   
+   FOREIGN KEY (meal_id) references `meal`(id)
+);
+
+						-- TABLE MEAL --
+-- Get all meals
+SELECT * FROM meal;
+
+-- Add a new meal
+INSERT INTO meal ( title, description, location, `when`, max_reservation, price, created_date)
+VALUES ('pizza', '4 cheese', 'city center', '2020-06-7', '5', 99.50, '2020-06-01');
+INSERT INTO meal ( title, description, location, `when`, max_reservation, price, created_date)
+VALUES ('lasagna', 'vegetarian', 'city center', '2020-06-7', '5', 75.50, '2020-06-01');
+
+-- Get a meal with any id, fx 1
+SELECT * FROM meal 
+WHERE id = 2;
+
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal SET description = 'spicy'
+WHERE id=2; 
+
+-- Delete a meal with any id, fx 1
+DELETE FROM meal 
+WHERE id = 1;
+
+						-- TABLE RESERVATION --
+-- Get all reservations
+SELECT * FROM reservation;
+
+-- Add a new reservation
+INSERT INTO reservation (number_of_guests, meal_id, created_date)
+VALUES ('3', '2', '2020-06-01');
+
+-- Get a reservation with any id, fx 1
+SELECT * FROM reservation
+WHERE id = 1;
+
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE reservation SET  number_of_guests = '4'
+WHERE id=1; 
+
+-- Delete a reservation with any id, fx 1
+DELETE FROM reservation 
+WHERE id = 1;
+
+						-- TABLE REVIEW--
+-- Get all reviews
+SELECT * FROM review;  
+
+-- Add a new review  
+INSERT INTO review (title, description, meal_id, stasr, created_date )
+VALUES ('perfect', 'meal was amaizing', '2', '5', '2020-06-01');                    
```

Comment:
I'm getting an error here: `Error Code: 1054. Unknown column 'stasr' in 'field list'`